packages/ember-repl/addon/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-repl",
-  "version": "3.0.0-beta.7",
+  "version": "3.0.0-beta.8",
```

Comment:
This was changed because I had to do a manual release a while back and forgot to commit the version bump